IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/DocSchema.xml

```diff
@@ -52,6 +52,8 @@
 				<DocDefinitionRef Name="IfcReal" UniqueId="53666384-7ea3-46d2-ba10-986fd16883d8" />
 				<DocDefinitionRef Name="IfcNamedUnit" UniqueId="04a55c7c-d817-4574-83c4-325bf83bbd76" />
 				<DocDefinitionRef Name="IfcLogical" UniqueId="155b841f-664f-4175-818f-326f654bf3fe" />
+				<DocDefinitionRef Name="IfcValue" UniqueId="ebb6f169-0bb0-4d4c-adf3-b93a7b706ca5" />
+				<DocDefinitionRef id="IfcMeasureValue_0zhETP7Hz2E94mtLjvV20s" Name="IfcMeasureValue" UniqueId="3dace759-1d1f-4238-9130-dd5b797c2036" />
```

Comment:
How come the rest have no `id` attribute?

Reply:
Good question. Have to look into it.

Reply:
Newly added get an id. Could change this but not priority.